dvc/repo/brancher.py

```diff
@@ -34,7 +34,7 @@ def brancher(  # noqa: E302
 
     scm = self.scm
 
-    self.tree = WorkingTree(self.root_dir)
+    self.tree = CleanTree(WorkingTree(self.root_dir))
```

Comment:
Guess need to wrap tree from down below too, right?

Reply:
> Guess need to wrap tree from down below too, right?

@pared ^

Reply:
@efiop Do you mean the one obtained with `get_tree`?

Reply:
@pared Yes 🙂 Will fix https://github.com/iterative/dvc/pull/2974/files#r360827242 too